MapboxNavigation/NavigationViewController.swift

```diff
@@ -165,15 +166,6 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
     public var routeController: RouteController!
     
     /**
-     `simulate` provides simulated location updates along the given route.
-     */
-    public var simulatesLocationUpdates: Bool = false {
```

Comment:
We can't do the same for ReplayLocationManager, so for consistency, I'd prefer not to expose this shorthand.